server.js

```diff
@@ -22,17 +22,32 @@ app.get('/location', (request, response) => {
   // normalizing data
   const jsonInfo = require('./data/location.json');
   const info = jsonInfo[0];
-  console.log(info);
 
   searchedLoc = request.query.city;
-  // 
+  // this works and I love it
```

Comment:
Great to see comments here. Additional specifics as to the success would be awesome for collaborating devs